react/__mocks__/vtex.styleguide.js

```diff
@@ -61,6 +61,27 @@ export const Input = forwardRef(function Input(
   )
 })
 
+export const InputSearch = forwardRef(function Input(
```

Comment:
```suggestion
export const InputSearch = forwardRef(function InputSearch(
```

Reply:
This change on vtex.styleguide.js doesn't affect anything so you can commit it.